src/routes/(auth)/signup/+page.svelte

```diff
@@ -23,17 +26,33 @@
             return;
         }
 
-        const userSpec: UserSpec = {
+        const userData = {
             firstName: firstNameInput.getValue(),
             lastName: lastNameInput.getValue(),
             email: emailInput.getValue(),
+            password: passwordInput.getValue(),
         };
-        const user = await BackendController.getInstance().createUser(
-            userSpec,
-            passwordInput.getValue(),
-        );
-        // TODO: Login and redirect to the home page
-        console.log(user);
+
+        BACKEND.register(userData)
+            .then((user) => {
+                const { accessToken, refreshToken } = user.response;
+                // TODO: Login with accessToken, store both tokens and redirect to the home page,
+                // will be completed in #134
+                console.log(
+                    `New user was registered with the following token: ${accessToken} (refreshToken: ${refreshToken})`,
+                );
+            })
+            .catch((error) => {
+                if (error.code == "ALREADY_EXISTS") {
+                    registrationError = {
+                        errorTitle: "An account with this email address already exists.",
+                        errorDetails: "Try logging in or resetting your password",
+                    };
+                    return;
+                }
+                registrationError = { errorTitle: "Something went wrong while registration." };
+                console.error(error);
```

Comment:
Shouldn't we log the other error as well?
```suggestion
                    registrationError = {
                        errorTitle: "An account with this email address already exists.",
                        errorDetails: "Try logging in or resetting your password",
                    };
                } else {
                    registrationError = { errorTitle: "Something went wrong while registration." };
                }
                console.error(error);
```

Reply:
Probably we could, yep. I didn't add it, as it will be displayed, but yes we should log every error as well.